modules/dists/ReplicatedDist.chpl

```diff
@@ -424,6 +424,16 @@ class ReplicatedArr : BaseArr {
         halt("locale ", here.id, " has no local replicand");
     return localArrs[here.id];
   }
+
+  /* Access another locale's local array representation by locale id */
+  proc replicand(id: here.id.type) ref {
```

Comment:
Field names can be nouns. Some methods are sortof accessors or similar to them. In that case I don't have a problem with using a noun. I think that `student.height()` fits into that category.

In the case of `replicand`, I don't have a problem with `A.replicand` because it's arguably accessing a property of A. What is some evidence that it's a "property" rather than an "action"?
 * it's accessing something fundamentally stored in the data structure (A replicated array might be viewed as consisting of replicands...)
 * It returns something that makes sense as the LHS of an assignment statement

In other words, if I was writing complete English sentences in my programming, `A.replicand` would be a *noun phrase* that I would presumably then want to do something else with using a verb. `A.replicand.split()` for an arbitrary example, has reasonable code-to-English properties.

Anyway for these reasons, I prefer it be called `replicand` in this case and not `getReplicand`.